Library/Formula/direnv.rb

```diff
@@ -23,8 +17,8 @@ def caveats; <<-EOS.undent
     At the END of your ~/.bashrc or ~/.zshrc, add the following line:
       eval "$(direnv hook $0)"
 
-    See the wiki for docs and examples:
-      https://github.com/zimbatm/direnv/wiki/
+    See the website for more docs and shell support:
+      http://www.direnv.net
```

Comment:
Ok, can we have `Configure direnv by reading:` and linking to a specific page? If there's no specific page or just the manpage I don't think that's useful; people will look at those things anyway.